packages/components/organisms/f-offers/src/store/offers.module.js

```diff
@@ -118,5 +96,28 @@ export default {
                 await dispatch(ACTION_HANDLE_CUSTOMER_DATA, { url, timeout });
             }
         }
+    },
+
+    mutations: {
+        [MUTATION_SET_AUTH_TOKEN]: (state, value) => {
+            state.authToken = value;
+        },
+
+        [MUTATION_SET_CUSTOMER_DATA]: (state, { friendlyName, firstName, lastName }) => {
+            state.customer = {
+                friendlyName,
+                firstName,
+                lastName
```

Comment:
If these are not defined in the payload, does that mean that the state values could be set from `null` to `undefined`? If so, please could we set default values as `null` to ensure that an 'unset' value remains predictably as null?

Reply:
again good point will address in follow up PR